install_config/configuring_azure.adoc

```diff
@@ -11,26 +11,49 @@
 toc::[]
 
 == Overview
-{product-title} can be configured to access an
-link:https://azure.microsoft.com/en-us/services/storage/disks/[Azure
+{product-title} can be configured to access
+link:https://azure.microsoft.com/en-us/services/storage/disks/[Microsoft Azure
 infrastructure], including
 xref:../install_config/persistent_storage/persistent_storage_azure.adoc#install-config-persistent-storage-persistent-storage-azure[using
-Azure disk as persistent storage] for application data. After Azure is
+Azure disk as persistent storage] for application data. After Microsoft Azure is
 configured properly, some additional configurations need to be completed on the
 {product-title} hosts.
 
 [[configuring-azure-permissions]]
 == Permissions
-Configuring Azure for {product-title} requires the following role:
+Configuring Microsoft Azure for {product-title} requires the following role:
 
 [cols='1,3']
 |===
 
 | Contributor
-| To create and manage all types of Azure resources.
+| To create and manage all types of Microsoft Azure resources.
 
 |===
 
+[[configuring-azure-prerequisites]]
+== Prerequisites
+
+* If you are using Microsoft Azure Disk as an OpenShift Persistent Volume for  {product-title} version 3.7 or lower, you must enable Azure Cloud Provider.
++
+[NOTE]
+====
+This is not required if you are using Azure File, Azure Blob storage for docker
+registry, or Azure Active Directory OpenID Connect for master authentication.
```

Comment:
I'm afraid this note is based on my comment. However, I found my comment is worng. Azure File requires Azure Cloud Provider too. Could you remove this note?